src/App/App.test.tsx

```diff
@@ -201,4 +201,29 @@ describe('App', () => {
     userEvent.click(screen.getByText('View Saved'))
     expect(screen.getByText('military western ska')).toBeInTheDocument()
   });
+
+  test('should return a failed message if fetch call for genres fails', async () => {
```

Comment:
It would be interesting to use a mockRejectedValue here, but this solves our testing nicely.